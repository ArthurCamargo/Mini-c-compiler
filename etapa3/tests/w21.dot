digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n1 -> n5;
n2;
n2 -> n6;
n2 -> n7;
n6;
n6 [label="i"];
n7;
n7 [label="0"];
n2 [label="="];
n3;
n3 [label="i"];
n4;
n4 -> n8;
n4 -> n9;
n8;
n8 [label="i"];
n9;
n9 [label="2"];
n4 [label="="];
n5;
n5 -> n10;
n5 -> n11;
n5 -> n12;
n10;
n10 [label="i"];
n11;
n11 [label="3"];
n12;
n12 -> n13;
n12 -> n14;
n13;
n13 [label="i"];
n14;
n14 [label="4"];
n12 [label="="];
n5 [label="="];
n1 [label="for"];
n0 [label="f"];

}