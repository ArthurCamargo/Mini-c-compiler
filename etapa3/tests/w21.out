0x55bf856e4350
0x55bf856e4350, 0x55bf856e48e0
0x55bf856e48e0
0x55bf856e48e0, 0x55bf856e44f0
0x55bf856e48e0, 0x55bf856e4560
0x55bf856e48e0, 0x55bf856e4670
0x55bf856e48e0, 0x55bf856e4780
0x55bf856e44f0
0x55bf856e44f0, 0x55bf856e4450
0x55bf856e44f0, 0x55bf856e44a0
0x55bf856e4450
0x55bf856e4450 [label="i"];
0x55bf856e44a0
0x55bf856e44a0 [label="0"];
0x55bf856e44f0 [label="="];
0x55bf856e4560
0x55bf856e4560 [label="i"];
0x55bf856e4670
0x55bf856e4670, 0x55bf856e45d0
0x55bf856e4670, 0x55bf856e4620
0x55bf856e45d0
0x55bf856e45d0 [label="i"];
0x55bf856e4620
0x55bf856e4620 [label="2"];
0x55bf856e4670 [label="="];
0x55bf856e4780
0x55bf856e4780, 0x55bf856e46e0
0x55bf856e4780, 0x55bf856e4730
0x55bf856e4780, 0x55bf856e4890
0x55bf856e46e0
0x55bf856e46e0 [label="i"];
0x55bf856e4730
0x55bf856e4730 [label="3"];
0x55bf856e4890
0x55bf856e4890, 0x55bf856e47f0
0x55bf856e4890, 0x55bf856e4840
0x55bf856e47f0
0x55bf856e47f0 [label="i"];
0x55bf856e4840
0x55bf856e4840 [label="4"];
0x55bf856e4890 [label="="];
0x55bf856e4780 [label="="];
0x55bf856e48e0 [label="for"];
0x55bf856e4350 [label="f"];
